apps/framework-docs/src/pages/building/capture-data.mdx

```diff
@@ -0,0 +1,55 @@
+import { Callout } from 'nextra/components'
+import { Tabs } from 'nextra/components'
+
+# Capture data
+MooseJS makes it easy to capture data from your user facing applications by providing typed SDKs which are automatically generated from your data models. 
+
+## From MooseJS SDKs
+<Callout type="info" emoji="ℹ️">
+  We currently support Typescript SDKs. Support for other languages is coming soon.
```

Comment:
Would be great to have a link to a feature tracker here so that people can upvote a language they would want to see.

Reply:
or maybe the slack so that they join to suggest changes